source/fundamentals/gridfs.txt

```diff
@@ -0,0 +1,707 @@
+.. _csharp-gridfs:
+
+=================
+Store Large Files
+=================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: binary large object, blob, storage, code example
+
+Overview
+--------
+
+In this guide, you can learn how to store and retrieve large files in
+MongoDB by using **GridFS**. GridFS is a specification implemented by 
+the {+driver-short+} that describes how to split files into chunks when storing them
+and reassemble those files when retrieving them. The driver's implementation of
+GridFS is an abstraction that manages the operations and organization of
+the file storage. 
```

Comment:
S: consider removing this sentence - it might confuse users to differentiate between GridFS spec vs implementation